src/Sulu/Bundle/ContentBundle/Resources/public/js/components/single-internal-link/main.js

```diff
@@ -185,46 +184,27 @@ define([], function() {
         /**
          * initialize column navigation
          */
-        overlayOkCallback = function () {
```

Comment:
We have talked about this for some time, and this is the most consistent solution. And even if we would change it, it shouldn't be done in this PR.